lib/bolt_spec/plans/mock_executor.rb

```diff
@@ -17,7 +17,7 @@ class UnexpectedInvocation < ArgumentError; end
 
     # Nothing on the executor is 'public'
     class MockExecutor
-      attr_reader :noop, :error_message, :in_parallel
+      attr_reader :noop, :error_message, :in_parallel, :transports
```

Comment:
Sorry, I feel like I'm missing something - where does this get called?

Reply:
Spec tests.